content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -392,6 +392,42 @@ pq.add(6);                      // [7, 6, 5]
 
 </JavaSection>
 
+<PySection>
+
+### [Python](https://docs.python.org/3/library/heapq.html)
+
+In Python, we delete and retrieve the element of **lowest** rather than highest priority.
```

Comment:
i mean maybe 0 means a high priority
just say in python, unlike java/cpp, the lowest # is removed first

Reply:
yeah